src/Aeon.Acquisition/SynchronizerMonitor.bonsai

```diff
@@ -111,7 +118,7 @@ Item2 as Names)</scr:Expression>
       <Expression xsi:type="scr:ExpressionTransform">
         <scr:Expression>new(
 Item1.Stats.Mean as MeanTimestamp,
-DateTime(1904, 1, 1) + TimeSpan.FromSeconds(Item1.Stats.Mean) as MeanUtcTimestamp,
+DateTime(1904, 1, 1) + TimeSpan.FromSeconds(Double.IsNan(Item1.Stats.Mean)?Item1.SyncTimestamp: Item1.Stats.Mean) as MeanUtcTimestamp,
```

Comment:
Can you add spaces between the `?` and `:` characters?

Basically do:
```
DateTime(1904, 1, 1) + TimeSpan.FromSeconds(Double.IsNan(Item1.Stats.Mean) ? Item1.SyncTimestamp : Item1.Stats.Mean) as MeanUtcTimestamp
```

Reply:
Fix in 8cd7b22a3c049ff5517f9fa27ffe188762d5e684